drivers/hmis/spec/requests/hmis/enrollment_last_contact_spec.rb

```diff
@@ -0,0 +1,175 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'rails_helper'
+require_relative 'login_and_permissions'
+require_relative '../../support/hmis_base_setup'
+
+RSpec.describe Hmis::GraphqlController, type: :request do
+  before(:all) do
+    cleanup_test_environment
+  end
+  after(:all) do
+    cleanup_test_environment
+  end
+
+  include_context 'hmis base setup'
+  include_context 'hmis service setup'
+
+  before(:each) do
+    hmis_login(user)
+  end
+
+  let(:query) do
+    <<~GRAPHQL
+      query EnrollmentsWithLastContactDate(
+        $clientId: ID!
+      ) {
+        client(id: $clientId) {
+          enrollments {
+            nodesCount
+            nodes {
+              id
+              lastContact {
```

Comment:
`drivers/hmis/spec/requests/hmis/client_enrollments_visibility_spec.rb` this spec tests `can_view_limited_enrollment_details` permission, maybe add this to the query in that test and assert its presence on limited-access enrollments ?